client/styles/pages/home.scss

```diff
@@ -6,6 +6,14 @@
         position: relative;
         padding: 0 20px 60px;
         width: 100%;
+        .pseudo-header {
```

Comment:
I would put this in its own scss file and import it into the `PseudoHeader` component.

Reply:
Good call, gonna separate out